pallets/moonbeam-lazy-migrations/src/lib.rs

```diff
@@ -106,5 +111,36 @@ pub mod pallet {
 			)
 			.into())
 		}
+
+		// TODO(alexandru): This extrinsic should be removed once Gov V1 is removed.
+		// Note: We don't need to unreserve any funds, as they are assumed to be already
+		// unreserved prior to this operation and the proposal submission disabled.
+		#[pallet::call_index(1)]
+		#[pallet::weight(
+			<T as frame_system::Config>::DbWeight::get()
+				.reads_writes((*limit + 1).into(), (*limit).into())
+		)]
+		pub fn unlock_democracy_funds(
+			origin: OriginFor<T>,
+			limit: u32,
+		) -> DispatchResultWithPostInfo {
+			ensure_signed(origin)?;
+			ensure!(limit != 0, "Limit cannot be zero!");
```

Comment:
Please never use raw string with ensure!, the raw string is skip by SCALE and we got a generic "Other" error message that is not helpful to debug

Reply:
What is the right approach, adding an error inside pallet and using that?

Should I change the one in `clear_local_assets_storage` as a drive-by too?

Reply:
> Should I change the one in clear_local_assets_storage as a drive-by too?

It was already fix by @RomarQ in #2561, I could not see it in its initial PR because it introduced this line after my approval and then merged directly.